<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SM4Class</class>
 <widget class="QMainWindow" name="SM4Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SM4文件加密系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="show_all_file_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>显示所有文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:14pt</string>
       </property>
       <property name="text">
        <string>文件数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="num_of_files_Lable">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:14pt;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_file_Edit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_file_Button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="all_files_ListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f0f0f0;padding-left:22px;padding-right:30px;color:#696969;padding-top:22px;font-size:22pt;font-family:consolas,&quot;STKaiti&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="gridSize">
     <size>
      <width>0</width>
      <height>44</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="change_loginPass_Action"/>
    <addaction name="delete_user_actions"/>
    <addaction name="separator"/>
    <addaction name="exit_Action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>加密</string>
    </property>
    <addaction name="encrypt_files_Action"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>解密</string>
    </property>
    <addaction name="decrypt_singleFile_Action"/>
    <addaction name="separator"/>
    <addaction name="decrypt_allFiles_Action"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>密钥管理</string>
    </property>
    <addaction name="change_key_Action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="encrypt_files_Action">
   <property name="text">
    <string>加密文件</string>
   </property>
  </action>
  <action name="actionAll_File">
   <property name="text">
    <string>All File</string>
   </property>
  </action>
  <action name="change_loginPass_Action">
   <property name="text">
    <string>修改登录密码</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="exit_Action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="decrypt_singleFile_Action">
   <property name="text">
    <string>选中文件</string>
   </property>
  </action>
  <action name="decrypt_allFiles_Action">
   <property name="text">
    <string>所有文件</string>
   </property>
  </action>
  <action name="change_key_Action">
   <property name="text">
    <string>更新密钥</string>
   </property>
  </action>
  <action name="delete_user_actions">
   <property name="text">
    <string>删除账户</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SM4.qrc"/>
 </resources>
 <connections/>
</ui>
